src/siphash.c

```diff
@@ -133,7 +133,7 @@ uint64_t siphash(const uint8_t *in, const size_t inlen, const uint8_t *k) {
     v0 ^= k0;
 
     for (; in != end; in += 8) {
-        m = U8TO64_LE(in);
+        memcpy(&m, &U8TO64_LE(in), sizeof(*in));
```

Comment:
`U8TO64_LE` uses unaligned access only on CPUs that support it. See (`UNALIGNED_LE_CPU`):
https://github.com/redis/redis/blob/e6478cfd10ae5ce8312c49823142d4dfcf0f45bb/src/siphash.c#L88-L96